doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
+        3.2 .............................. |xcodebuild_xcpretty_flags|
+        3.3 .............................. |xcodebuild_xcpretty_testing_flags|
 
 ==============================================================================
-INTRODUCTION                           *xcodebuild*
+ABOUT (1)                                                           *XCB-About*
 
-Xcodebuild.vim is a thin wrapper around Apple's xcodebuild command line
-application that lets you build and test your Xcode projects from within vim.
+`xcodebuild.vim` is a thin wrapper around Apple's `xcodebuild` command line
+application that lets you build and test your Xcode projects from within Vim.
 It dynamically figures out your project information and passes the proper
-flags to xcodebuild.
+flags to `xcodebuild`.
+
+If you have `xcpretty`[1] installed, `xcodebuild.vim` will use it to reformat
+the output from `xcodebuild`.
+
+This plugin was written by Gordon Fontenot[2]. Bugs and feature requests are
+welcomed, and can be posted on the GitHub repo[3].
 
-If you have xcpretty installed, it will use that to reformat the output from
-xcodebuild. By default, xcodebuild.vim passes the `--color` flag to xcpretty.
+[1]: https://github.com/supermarin/xcpretty
+[2]: http://gordonfontenot.com
+[3]: https://github.com/gfontenot/vim-xcodebuild
 
 ==============================================================================
-CONFIGURATION                             *xcodebuild-configuration*
+USAGE (2)                                                           *XCB-Usage*
+
+`xcodebuild.vim` provides two commands, one for building the project, and one
+for testing the project. It dynamically looks at your project configuration
+and determines which flags to pass to `xcodebuild`.
+
+Currently, `xcodebuild.vim` only looks at the first scheme that it finds in
+the `xcodeproj` file in the working directory. It uses the build settings in
+that scheme to determine which sdk to test against. If you have an
+`xcworkspace` file that contains your project (as is common with projects
+using CocoaPods[4]), it will build the scheme through the workspace.
+
+[4]: https://cocoapods.org/
+
+------------------------------------------------------------------------------
+                                                                       *XBuild*
+2.1 XBuild~
+
+Build the project with `xcodebuild`.
+
+The first time this command is run, there will be a small delay as we parse
+the scheme information from the project. This scheme info is cached with your
+Vim session, so subsequent runs will execute faster.
+
+------------------------------------------------------------------------------
+                                                                       *XTest*
+2.2 XTest~
+
+Run the project tests through `xcodebuild`.
+
+The first time this command is run, there will be a small delay as we parse
+the sdk information for the scheme, and might be a delay while we parse the
```

Comment:
sdk -> SDK

and nix the "we"  here too

Unless you come over to every user's house and type for them? If so, I'd add another section to this help document listing legal disclaimers.